en/docs/learn/configuring-oauth2-openid-connect.md

```diff
@@ -30,6 +30,11 @@ the users with an authorization server-based authentication.
     **OAuth2/OpenID Connect Configuration** form.  
     ![oauth2-openid-connect-configuration](../assets/img/tutorials/oauth2-openid-connect-configuration.png)
     
+    !!! Note
+    WSO2 Identity server supports RP-initiated logout requests to OIDC IDPs. This feature is supported with the
+     update-0147. 
+    ![oauth2-openid-connect-federated-logout](../assets/img/tutorials/oidc-federated-logout.png)
```

Comment:
!!! note
     WSO2 Identity Server supports RP-initiated logout requests to OpenID Connect identity providers.

     To use this feature, apply the **0147** WUM update for WSO2 Identity Server 5.9.0 using the WSO2 Update 
     Manager (WUM). To deploy a WUM update into production, you need to have a paid subscription. If you do not 
     have a paid subscription, you can use this feature with the next version of WSO2 Identity Server when it is 
     released. For more information on updating WSO2 Identity Server using WUM, see Getting Started with WUM.

Reply:
Please also make sure the content is properly indented inside the note as follows;

!!! note
     <content>